lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -2941,6 +2943,56 @@ SwiftLanguageRuntimeImpl::GetValueType(ValueObject &in_value,
   return Value::ValueType::Scalar;
 }
 
+namespace {
+struct SwiftNominalType {
+  std::string module;
+  std::string identifier;
+};
+
+std::optional<SwiftNominalType> getSwiftClass(ValueObject &valobj,
```

Comment:
Nit: capitalize this (`GetSwiftClass`) since it's the standard for lldb.

Reply:
thanks! done